src/basis/data.js

```diff
@@ -2481,6 +2487,9 @@
       var deleted = delta.deleted;
       var cache = eventCache[datasetId];
 
+      if (cache === PREVENT_ACCUMULATIONS)
```

Comment:
In this case no event dispatching at all. `cache === PREVENT_ACCUMULATIONS` should be added to next `if` statement expression.